tests/unit/functions.tcl

```diff
@@ -631,10 +631,10 @@ start_server {tags {"scripting"}} {
 
     test {LIBRARIES - register function inside a function} {
         r function load LUA lib {
-            redis.register_function(
+            library.register_function(
```

Comment:
@yossigo maybe we should re-consider the API name.
it can still be a different object at each time, but it can have the same name so that the user won't know he's working on a different API object.
do you think the new name serves us to avoid exposing the wrong APIs, or also serves the user to understand he's working with a different set of APIs?